src/Aspire.Hosting.Azure/build/Aspire.Hosting.Azure.targets

```
@@ -0,0 +1,6 @@
+<Project>
+  <ItemGroup>
+    <JsonSchemaSegment Include="$(MSBuildThisFileDirectory)..\AspireAzureConfigurationSchema.json"
+                       FilePathPattern="appsettings\..*json" />
```

A lot of times these properties go into "User Secrets", which is a `secrets.json` file. This is a Regex, so I think we should also add `secrets.json` to it. @alexgav - did we ever hook up the FilePathPattern correctly in VS? 

```suggestion
                       FilePathPattern="appsettings\..*json|secrets.json" />
```

---

I tried to add it, but it didn't work for me. So I skipped for now